app/models/distribution.rb

```diff
@@ -9,6 +9,45 @@ class Distribution < BaseModel
   has_many :audio_version_templates, through: :distribution_templates
   serialize :properties, HashSerializer
 
+  def self.check_published!(data = nil)
+    data ||= {}
+    from_secs = (data[:from_secs] || '60').to_i
```

Comment:
a nit: `from` and `to` are a little confusing for me here as variables names. You're defining a range window within which you want to pull recently published stories? Maybe `start` and `end` would be clearer? I find naming hard when doing Time math, so maybe it's just me who is confused.

Reply:
`end` is a reserved word in ruby so I try to avoid that, but I could use `start_time` and `end_time`?

Reply:
ok, changed